front/js/shared/services/tracking.es6

```diff
@@ -8,13 +8,13 @@
   window.dataLayer = window.dataLayer || [];
   // Load ecommerce plugin for gaTracker
   try {
-    ga('require', 'ecommerce');
+    ga('require', 'ecommerce');  // Ignore ESLintBear (no-use-before-define)
   } catch (e) {
-    var ga = console.error;
+    const ga = console.error;  // Ignore ESLintBear (no-unused-vars)
```

Comment:
This variable prevents an error in `GATracker` initialization, so we should use `var` here

```js 
{const a = 1}
undefined
a
ReferenceError: a is not defined[Подробнее] debugger eval code:1:1
{var a = 1}
undefined
a
1
```

Reply:
@artemiy312 , wow, i'll try to play with var's context somehow